core/src/main/java/org/apache/iceberg/encryption/PlaintextEncryptionManager.java

```diff
@@ -18,15 +18,19 @@
  */
 package org.apache.iceberg.encryption;
 
-import java.nio.ByteBuffer;
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.OutputFile;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class PlaintextEncryptionManager implements EncryptionManager {
+  private static final EncryptionManager INSTANCE = new PlaintextEncryptionManager();
```

Comment:
Can you also deprecate the public constructor so that we can remove it in 1.6.0?